engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -403,8 +406,13 @@ namespace dmGameSystem
             item.m_Model = resource->m_Meshes[i].m_Model;
             item.m_Mesh = resource->m_Meshes[i].m_Mesh;
             item.m_MaterialIndex = resource->m_Meshes[i].m_Mesh->m_MaterialIndex;
+            item.m_AabbMin = item.m_Mesh->m_AabbMin;
+            item.m_AabbMax = item.m_Mesh->m_AabbMax;
             item.m_Enabled = 1;
             component->m_RenderItems.Push(item);
+
+            // printf("AABB: (%f, %f, %f)  (%f, %f, %f)\n", item.m_AabbMin.getX(), item.m_AabbMin.getY(), item.m_AabbMin.getZ(),
+            //         item.m_AabbMax.getX(), item.m_AabbMax.getY(), item.m_AabbMax.getZ());
```

Comment:
Maybe wrap this is in a pragma or add a comment or something if it's useful for debugging